libs/util/pipes/src/lib/extend-date-pipe.ts

```diff
@@ -7,6 +7,7 @@ import { DatePipe } from '@angular/common';
 export class ExtendDatePipe implements PipeTransform {
   readonly customFormats: { [type: string]: string } = {
     medium: 'EEEE, d MMM y, HH:mm',
+    short: 'd MMM, HH:mm',
```

Comment:
This is conflicting with https://github.com/heddendorp/tumi-app/blob/refactor/nx/libs/util/pipes/src/lib/extend-date.pipe.ts#L10
Please resolve before merging